request-filterer/request-filterer-store/src/test/java/com/duckduckgo/request/filterer/store/RealCookiesFeatureToggleRepositoryTest.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.request.filterer.store
+
+import com.duckduckgo.request.filterer.api.RequestFiltererFeatureName.RequestFilterer
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.verify
+
+class RealCookiesFeatureToggleRepositoryTest {
+    lateinit var testee: RequestFiltererFeatureToggleRepository
+
+    private val mockCookiesFeatureToggleStore: RequestFiltererFeatureToggleRepository = mock()
```

Comment:
Variables in this file should be `RequestFilterer` instead of `Cookies`

Reply:
Done